src/voipms.ts

```diff
@@ -63,12 +60,14 @@ export class Client extends EventEmitter {
   }
 
   async sendSMS(dst: string, message: string) {
```

Comment:
can we rename this to `sendMessage` just to be consistent?

Reply:
Good point, done !